ggshield/core/clickutils/default_command_group.py

```diff
@@ -0,0 +1,37 @@
+from typing import Any, List
+
+import click
+
+
+# Use https://github.com/click-contrib/click-default-group instead?
```

Comment:
@agateau-gg I could use your help on this one.

I don't think it is a good idea to use the lib because it adds a dependency for a very small code snippet that hasn't changed for years.

However, I think it is a good idea to keep the reference for documentation and to find help if we have an issue with it.